### PR TITLE
fix(Settings): listening to all keystrokes while taking link input

### PR DESCRIPTION
**Pull request by aakash19here**
## Related Issue
https://github.com/typehero/typehero/issues/669

## Screenshots/Video (if applicable):

https://github.com/typehero/typehero/assets/86520455/b6c1bb12-2578-43b9-8d69-549a3438bef3

### CODE REVIEW

File: apps/web/src/components/settings/index.tsx [hunk 1]
```diff
@@ -105,6 +106,7 @@ export function Settings({ profileData, username }: { profileData: FormSchema; u
                       defaultValue={val.url}
                       placeholder="Link to social profile"
                       {...register(`userLinks.${i}.url`)}
+                      onChange={() => setFocus(`userLinks.${i}.url`, { shouldSelect: true })}
```
Comment (bautistaaa) [inline apps/web/src/components/settings/index.tsx:109]: you shouldnt have to setfocus on a keystroke

Comment (aakash19here) [reply apps/web/src/components/settings/index.tsx:109]: - Tried wrapping it with Controller component from react-hook-form , didn't worked their 
- Tried changing the key of FormItem component for not causing re-render , it didn't help either 
Can you please suggest something

**Comment by bautistaaa** [comment]
fixed in main via million ignore

**Comment by aakash19here** [comment]
Ohhh thanks trash <3

**Pull request closed without merging**